### DISCUSSION TITLE
Wire:Model Issue V3

### DISCUSSION THREAD

**Discussion by robs007** [livewire/livewire#8964] [1. Help] [answered]
I'm having an issue with wire:model in V3. I have used this approach multiple times without issue. I'm sure I'm just missing something. I can't seem to get a model:wire to work on a input element using a model as the property. I believe it should work with student.last_name. 
@json shows the model, so I know it's loading on the view.
{{$student->std_last_name}} works.
If I add $this->lastname = $this->student->std_last_name to mount, it works, but I don't think I should have to do that.
If I use a single property $name = 'John' it works.
 In V2 this works fine. I must be missing something simple.

`View - <input type="text" wire:model="student.std_last_name"  clearable class="min-w-full"/>`

`use App\Models\ResStudent;`

` public ResStudent $student;`

  public function mount(): void
    {
        $this->student = ResStudent::find(13052);
   }

    public function rules()
    {
        return [
            'student.std_last_name' => 'required'|'string',

        ];
    }

**Comment by joshhanley** [comment, marked as answer]
@robs007 Please read the upgrade guide in the docs, it should cover model binding. If you're still having trouble after that, then let us know 🙂
